packages/Button/src/Button.tsx

```diff
@@ -407,7 +405,7 @@ export default class Button extends Component<ButtonProps> {
     return (
       <button
         type="button"
-        {...rest as HTMLElementProps<'button'>}
+        {...(rest as HTMLElementProps<'button'>)} //eslint-disable-line
```

Comment:
What was eslint complaining about here?

Reply:
Was adding the () and was just testing if that was what was breaking the test suite... tried everything before adding size/variant to the test files